optaplanner-core/src/main/java/org/optaplanner/core/config/localsearch/decider/acceptor/AcceptorConfig.java

```diff
@@ -495,6 +524,27 @@ public Acceptor buildAcceptor(HeuristicConfigPolicy configPolicy) {
                     stepCountingHillClimbingSize_, stepCountingHillClimbingType_);
             acceptorList.add(acceptor);
         }
+        if ((acceptorTypeList!= null && acceptorTypeList.contains(AcceptorType.GREAT_DELUGE))
+                || greatDelugeRainSpeed != null || greatDelugeRainSpeedRatio != null) {
+            GreatDelugeAcceptor acceptor = new GreatDelugeAcceptor();
+
+            if (greatDelugeRainSpeed != null && greatDelugeRainSpeedRatio != null) {
+                throw new IllegalArgumentException("You can use only rainSpeed or rainSpeedRatio at a time.");
```

Comment:
Let's not blame the user, so no "You ...". Let's keep it consistent with the other exceptions thrown when 2 fields are used together that can't, some something like "The greatDelugeRainSpeed (" + greatDelugeRainSpeed ") and ... cannot be both non null.